### ISSUE TITLE
Missing client_secret.json

### ISSUE THREAD

**Issue by IInvocation** [gilesknap/gphotos-sync#248]
**Describe the bug**
I'm trying to run gphotos-sync using docker-compose - but i just can't make the container find the client_secret.json. (Complains about a missing file "/.config/gphotos-sync/client_secret.json") - although it's there. (I copied the file (root-owned) to:
- ${USERDIR}/docker/gphotos
- ${USERDIR}/docker/gphotos/gphotos-sync

**To Reproduce**
my compose-file (v2.3) looks like this:
  gphotos:
    image: gilesknap/gphotos-sync:latest
    container_name: gphotos-sync
    volumes:
      - ${USERDIR}/docker/gphotos:/config
      - /mnt/storage/Pictures/GSync/Sync:/storage
    command: /storage
    restart: always

** What i tried yet **
- passing user:(PUID):(PGID) in the compose-file (not working) after owning the file as my normal user

In short - what am i missing here? Also - as far as i understand:
/config should get mapped to /root/.config looking at the docker-file
Why is the software looking into a subfolder (gphotos-sync) for the client_secret.json? (Going by the readme - i shouldnt need that subfolder)

**Comment by oweitman** [comment]
di you entered the secret code from google to the application?
if not see
https://github.com/gilesknap/gphotos-sync/issues/243

**Comment by IInvocation** [comment]
I downloaded the file from googles dev-console, renamed it and copied to the mentioned locations.
The file contects look like valid json and contain the secret.

**Comment by oweitman** [comment]
this is only the first step. if you start the docker container you find in the log file a google link.
you have to enter this link in a browser annd you get a secret. this secret you have to enter in the script.
please read my step by step description in #243.

**Comment by IInvocation** [comment]
Hm - this is rather strange. I inspected the log (even after recreating and deleting everything) and starting again - it doesnt contain any link (using docker compose). However - i now used the docker run command (same volumes) - and instantly got the link in the output. By now it's busily downloading. I think this issue could be closed - though it's a little disturbing it won't work for docker-compose.

**Comment by gilesknap** [comment]
Thanks @oweitman. I think the issue is that the first run must be interactive for Google security reasons, so I'm closing this as a duplicate of #243
